gammapy/maps/wcs/tests/test_geom.py

```diff
@@ -620,3 +620,13 @@ def test_wcs_geom_to_even_npix():
     geom_even = geom.to_even_npix()
 
     assert geom_even.data_shape == (4, 4)
+
+
+def test_wcs_geom_no_zero_shape_cut():
+    geom = WcsGeom.create(skydir=(0, 2.5), binsz=(360, 5), width=(360, 180))
+    skydir = SkyCoord(110.0, 75.0, unit="deg", frame="icrs")
+
+    geom_cut = geom.cutout(skydir, width=5)
+
+    assert geom_cut.data_shape != (1, 0)
```

Comment:
why the two tests?

Reply:
I added this one such as it is more explicit that what we want to avoid in this test is the zero shape.